cartography/intel/gcp/cloudrun.py

```diff
@@ -610,7 +610,7 @@ def cleanup_gcp_cloudrun(neo4j_session: neo4j.Session, common_job_parameters: Di
 @timeit
 def sync(
     neo4j_session: neo4j.Session, cloudrun: Resource, project_id: str, gcp_update_tag: int,
-    common_job_parameters: Dict,
+    common_job_parameters: Dict, regions: list
```

Comment:
how are we using regions in cloudrun? i don't see any code for cloudrun using region parameter.

Reply:
for cloudrun there is no any attribute to find out the region of cloudrun Resources so for there we used the by default  `global` as as region so there is no need of filtering the data with regions because all resources has `region` attribute is `global`